src/DependencyResolver/Package.php

```diff
@@ -74,17 +75,15 @@ public static function fromComposerCompletePackage(CompletePackageInterface $com
             ? $phpExtOptions['build-path']
             : null;
 
-        /** @var list<string> $compatibleOsFamilies */
         $compatibleOsFamilies = $phpExtOptions !== null && array_key_exists('os-families', $phpExtOptions)
             ? $phpExtOptions['os-families']
-            : [];
+            : null;
 
-        /** @var list<string> $incompatibleOsFamilies */
         $incompatibleOsFamilies = $phpExtOptions !== null && array_key_exists('os-families-exclude', $phpExtOptions)
             ? $phpExtOptions['os-families-exclude']
-            : [];
+            : null;
```

Comment:
Could just be this no?

`$incompatibleOsFamilies = $phpExtOptions['os-families-exclude'] ?? null;`

Reply:
:heavy_check_mark: